### PR TITLE
Cleanup more weird Git edge-cases.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Purge the GC log and PR locks on more repositories if they exist.

### CODE REVIEW

File: cmd/brew-test-bot.rb [hunk 1]
```diff
@@ -782,6 +782,10 @@ def cleanup_shared
       end
 
       Pathname.glob("#{HOMEBREW_LIBRARY}/Taps/*/*").each do |git_repo|
+        pr_locks = "#{@repository}/.git/refs/remotes/*/pr/*/*.lock"
+        Dir.glob(pr_locks) { |lock| FileUtils.rm_rf lock }
+        FileUtils.rm_rf "#{@repository}/.git/gc.log"
```
Comment (ilovezfs) [inline cmd/brew-test-bot.rb]: not `rm_f`?

**Comment by ilovezfs** [comment]
Is this needed for Homebrew/brew, too?
